### PR TITLE
docs: added tutorial on export

### PR DESCRIPTION
**Pull request by PierreLeveau**
Here is the tutorial preview:
https://github.com/kili-technology/kili-python-sdk/blob/2ce38d83d98e4faf12aa342269d98a2a4c761863/docs/sdk/tutorials/export_a_kili_project.md

### CODE REVIEW

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+{'latestLabel': {'jsonResponse': {'CLASSIFICATION_JOB': {'categories': [{'name': 'VEHICLE'}]}}}, 'externalId': '0'}
+```
+ 1. You can now get your label this way, and for example write the category name into a `0.txt` file:
+```python
+for asset in assets:
+    if asset["latestLabel"]: # covers the assets without label
+        class_ = asset["latestLabel"]["jsonResponse"]["CLASSIFICATION_JOB"]["categories"][0]["name"]
+        with (Path("/tmp") / (asset["externalId"] + ".txt")).open("w", encoding="utf-8") as f:
+            f.write(class_)
+```
```
Comment (Jonas1312) [inline docs/sdk/tutorials/export_a_kili_project.md:42]: The enumerated list is not incremented

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+
+### Filtering specific labels per asset through the method filters
+You can specify label filters directly in the `.assets` and the `.labels` methods. The available filters are listed in the arguments
+of [`.assets`](https://python-sdk-docs.kili-technology.com/latest/sdk/asset/#kili.queries.asset.__init__.QueriesAsset.assets) and [`.labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.labels) methods.
+
+Once done, you can write the conversion code to obtain the data in the format you need.
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:49]: Once -> When. We should probably avoid ambiguous words

Comment (xczuba) [reply docs/sdk/tutorials/export_a_kili_project.md:49]: in the format you need -> in the format that you need. More precise this way.

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+**Get all the assets, but each will only have the labels with a consensus mark above 0.7**
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:51]: Get only the assets that have labels with a consensus mark above 0.7

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+```python
+assets = kili.assets("<your_project_id>", fields=["externalId", "labels.jsonResponse"], label_consensus_mark_gt=0.7)
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:53]: <your_project_id>  QA automation: we should probably replace items like this with variables.

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+# + asset conversion code
+```
+
+**Get all the labels with a consensus mark above 0.7**
+```python
+labels = kili.labels("<your_project_id>", fields=["labelOf.externalId", "jsonResponse"], consensus_mark_gt=0.7)
+# + label conversion code
+```
+
+**Get all the labels done by a given author**
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:63]: suggestion:
by a given author -> by a specific project member

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+```python
+labels = kili.labels("<your_project_id>", fields=["labelOf.externalId", "jsonResponse"], author_in=["John Smith"])
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:65]: QA automation: looks like we'd have to also simulate this user somehow. If Kili client return an error this is probably our only option. If it returns a blank list, we're probably ok...

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+        for label in asset["labels"]:
+            if label["isLatestReviewLabelForUser"] and "JOB_0" in label["jsonResponse"]:
+                annotation = label["jsonResponse"]["JOB_0"]
+                with (Path("/tmp") / (asset["externalId"] + ".json")).open("w", encoding="utf-8") as f:
+                    f.write(json.dumps(annotation))
+                break # once we find a latest label done by a reviewer, we move on to the next asset.
+```
+
+## Exporting a whole project
+There is also a method to export a full project into specific export formats. It can be useful when your goal is to use a standard export format.
```
[the remark this answers is not in the document]
Comment (xczuba) [reply docs/sdk/tutorials/export_a_kili_project.md:87]: a standard output format -> one of the standard output formats

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+| YOLO V7       | ❌   | ✅             | ✅                      |
+| Pascal VOC    | ✅   | ❌             | ❌                      |
+| COCO          | ❌   | ✅             | ✅                      |
+
+
+### The `.export_labels` method
+
+The `.export_labels` method enables the export of a full project. It does the following preprocessing:
+
+* Only fetches the labels of types `"DEFAULT"` and `"REVIEW"`.
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:108]: Maybe we could direct them to the explanation of types in docs: 
https://docs.kili-technology.com/docs/asset-lifecycle#label-types-and-definitions-throughout-an-asset-lifecycle

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+* If specified, only selects a subset of asset ids.
+* You can specify a standard export format (only available for a restricted set of ML task).
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:110]: You can specify a standard export format -> Exports labels to one of the standard formats

This will make it grammatically in line with the other points

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+```python
+from kili.client import Kili
+kili = Kili()
+kili.export_labels(
+    project_id = "<your_project_id>",
+    filename = "/tmp/export.zip",
+    fmt = "kili",
+)
+```
+will export all the assets payloads and the associated labels, with one json file per asset, into the `/tmp/export.zip` folder.
```
[the remark this answers is not in the document]
Comment (xczuba) [reply docs/sdk/tutorials/export_a_kili_project.md:146]: assets payloads -> asset payload

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+from kili.client import Kili
+kili = Kili()
+kili.export_labels(
+    project_id = "<your_project_id>",
+    filename = "/tmp/export.zip",
+    fmt = "kili",
+    single_file = True,
+)
+```
+will export all the assets payloads and the associated labels, with one json file per asset, into the `/tmp/export.zip` folder.
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:159]: Again, would move it before code, as an introductory sentence

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+
+### YOLO formats
+You can also export to the yolo format, when you have at least one Object Detection job with bounding boxes. You can select the `"yolo_v4"`, `"yolo_v5"` or `"yolo_v7"`. The difference between each format is the structure of the metadata YAML file, which specifies the object classes. In all the cases, it produces one file per asset in the Yolo format, containing the last DEFAULT or REVIEW label that has been produced. Each YOLO label has the following shape:
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:162]: "you can select"
comment: you don't really select. I think we need to phrase it differently, I think. Maybe "choose"?

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 13]
```diff
@@ -0,0 +1,207 @@
...
+```
+2        0  123 33 145
+^        ^  ^   ^  ^
+class    x  y   w  h
+```
+where:
+
+   * `class` is the class index in the classes list contained in the YOLO metadata file.
+   * `x` is the x-coordinate relative to the image width (between 0.0 and 1.0) of the center of the bounding box.
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:171]: Maybe we can add a short example? This is pretty vague. Applies to all of these lines

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 14]
```diff
@@ -0,0 +1,207 @@
...
+   * `y` is the y-coordinate relative to the image height (between 0.0 and 1.0) of the center of the bounding box.
+   * `w` is the width relative to the image width (between 0.0 and 1.0) of the bounding box.
+   * `h` is the height relative to the image height (between 0.0 and 1.0) of the bounding box.
+
+Here is how to call the YOLO export (YOLO V5 here):
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:176]: I'd rephrase. Call is pretty code-centric. How about: "Here's how to export to YOLO (in this example, YOLOv5):"

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 15]
```diff
@@ -0,0 +1,207 @@
...
+kili = Kili()
+kili.export_labels(
+    project_id = "<your_project_id>",
+    filename = "/tmp/export.zip",
+    fmt = "yolo_v5",
+    single_file = True,
+)
+```
+
+Please note that a standard YOLO file format should also include the path root to the assets, and also the `train`, `val` and `test` subfolders. Since this is up to the ML engineer or Data scientist which data goes where, we do not provide this layout.
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:189]: "Should" tends to be misleading. If it's something that must happen, we need to use "must".

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 16]
```diff
@@ -0,0 +1,207 @@
...
+
+
+### COCO format
+To export your data into the COCO format, you can do the following:
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:193]: "you can do the following" -> "run the following code"

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 17]
```diff
@@ -0,0 +1,207 @@
...
+
+## Preliminary steps
+ 1. Fetch the project ID from the Kili UI (in Settings / Admin):
+   ![Get Project ID from UI](../../assets/get_project_id.jpg)
+ 2. Ensure that your Kili API key as been set as an environment variable:
+ ```bash
+ export KILI_API_KEY = <YOUR_API_KEY>
+ ```
+ 3. Import packages and initialize the Kili client:
+```python
```
Comment (xczuba) [inline docs/sdk/tutorials/export_a_kili_project.md:16]: Don't we want to add a line of code somewhere that actually installs kili?

```
!pip install --upgrade kili
```

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 18]
```diff
@@ -0,0 +1,207 @@
+# How to export a Kili project
+With Kili, once you have annotated enough assets, you can export the data programmatically to train a machine learning algorithm from it. There are several ways to do it:
+
+ * fetch the assets and/or the labels one by one using `.assets` or `.labels`, perform the data transformation yourself, then write them to one or several output files.
+ * export the whole project as a dataset using `.export_labels`, which will create an archive containing one or several files containing the labels in the format you need.
+
+
+## Preliminary steps
+ 1. Fetch the project ID from the Kili UI (in Settings / Admin):
```
[the remark this answers is not in the document]
Comment (PierreLeveau) [reply docs/sdk/tutorials/export_a_kili_project.md:9]: discussed irl, no need to add the project creation

File: docs/sdk/tutorials/export_a_kili_project.md (hunk 4 above)
Comment (PierreLeveau) [reply docs/sdk/tutorials/export_a_kili_project.md:61]: to be done when we will migrate towards the testable solution

File: docs/sdk/tutorials/export_a_kili_project.md (hunk 6 above)
Comment (PierreLeveau) [reply docs/sdk/tutorials/export_a_kili_project.md:73]: to be done in doc automated QA

File: docs/sdk/tutorials/export_a_kili_project.md (hunk 13 above)
Comment (PierreLeveau) [reply docs/sdk/tutorials/export_a_kili_project.md:180]: I have added an image to illustrate that.

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 19]
```diff
@@ -0,0 +1,226 @@
...
+ * Fetch the assets and/or the labels one by one using `.assets` or `.labels`, perform the data transformation yourself, then write the data to one or several output files.
+ * Export the whole project as a dataset. To do that, use the `.export_labels` method that creates an archive containing the labels in your chosen format.
+
+
+## Preliminary steps
+ 1. Fetch the project ID from the Kili UI (in Settings / Admin):
+   ![Get Project ID from UI](../../assets/get_project_id.jpg)
+ 2. Ensure that your Kili API key as been set as an environment variable:
+ ```bash
+ export KILI_API_KEY = <YOUR_API_KEY>
```
Comment (Jonas1312) [inline docs/sdk/tutorials/export_a_kili_project.md:17]: remove spaces around =

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 20]
```diff
@@ -0,0 +1,226 @@
...
+ ```
+ 3. Install Kili if it has not been done already.
+```bash
+!pip install --upgrade kili
```
Comment (Jonas1312) [inline docs/sdk/tutorials/export_a_kili_project.md:21]: we can remove "!"

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 21]
```diff
@@ -0,0 +1,226 @@
...
+                    f.write(json.dumps(annotation))
+                break # once we find a latest label done by a reviewer, we move on to the next asset.
+```
+
+## Exporting a whole project
+There is also a method to export the whole project into specific export formats. It can be useful when your goal is to use one of the standard output formats.
+
+### Available formats
+
+Here are the available formats:
```
Comment (Jonas1312) [inline docs/sdk/tutorials/export_a_kili_project.md:99]: we can remove this line since the header above says it already

**Review by Jonas1312 — CHANGES_REQUESTED** [review]

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 22]
```diff
@@ -0,0 +1,224 @@
+# How to export a Kili project
+## Outline
+This tutorial explains the multiple ways to export a Kili project. It describes a per-label method involving label filtering and conversion, and also the solutions performing a full-project export. The methods are illustrated with code snippets.
+
+## Export methods
+With Kili, once you have annotated enough assets, you can export the data programmatically to train a machine learning algorithm with it. There are several ways to do it:
+
+ * Fetch the assets and/or the labels one by one using `.assets` or `.labels`, perform the data transformation yourself, then write the data to one or several output files.
+ * Export the whole project as a dataset. To do that, use the `.export_labels` method that creates an archive containing the labels in your chosen format.
```
Comment (Jonas1312) [inline docs/sdk/tutorials/export_a_kili_project.md:9]: add links to sdk doc

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 23]
```diff
@@ -0,0 +1,224 @@
...
+    if asset["latestLabel"]: # covers the assets without label
+        class_ = asset["latestLabel"]["jsonResponse"]["CLASSIFICATION_JOB"]["categories"][0]["name"]
+        with (Path("/tmp") / (asset["externalId"] + ".txt")).open("w", encoding="utf-8") as f:
+            f.write(class_)
+```
+
+
+### Filtering specific labels per asset through the method filters
+You can specify label filters directly in the `.assets` and the `.labels` methods. The available filters are listed in the arguments
+of [`.assets`](https://python-sdk-docs.kili-technology.com/latest/sdk/asset/#kili.queries.asset.__init__.QueriesAsset.assets) and [`.labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.labels) methods.
```
Comment (Jonas1312) [inline docs/sdk/tutorials/export_a_kili_project.md:55]: links to doc

File: docs/sdk/tutorials/export_a_kili_project.md [hunk 24]
```diff
@@ -0,0 +1,224 @@
...
+This will create an archive containing both:
+
+ * The COCO annotation file.
+ * A folder `data/` with all the assets.
+
+## Summary
+In this tutorial, we have seen several ways to export labels from a Kili project:
+
+* Using `.assets` and `.labels` and their filtering arguments, a subset of assets or labels can be selected and then exported.
+* Using `.export_labels`, the whole project can be exported into a standard output format.
```
Comment (Jonas1312) [inline docs/sdk/tutorials/export_a_kili_project.md:224]: links

**Review by Jonas1312 — CHANGES_REQUESTED** [review]
LGTM after those changes
